kmir/k-src/mir-rvalue.md

```diff
@@ -0,0 +1,371 @@
+```k
+require "mir-syntax.md"
+require "mir-configuration.md"
+require "mir-types.md"
+```
+
+Syntax of rvalues
+-----------------
+
+Rvalues are expression that appear on a right-hand-side of an assignment statement.
+
+```k
+module MIR-RVALUE-SYNTAX
+  imports BOOL
+  imports UNSIGNED-INT-SYNTAX
+  imports MIR-TYPE-SYNTAX
+  imports MIR-PLACE-SYNTAX
+  imports MIR-CONSTANT-SYNTAX
+```
+
+### [`Operand`](https://doc.rust-lang.org/beta/nightly-rustc/rustc_middle/mir/enum.Operand.html)
+
+Operands are leafs, i.e. the "basic components" of rvalues: either a loading of a place, or a constant.
+
+```k
+  syntax Operand ::= Place
```

Comment:
In `rust-lang` implmentation, there is this `Copy` operand. I guess it is implemented here. 
What is the intuition of dropping the `Copy`command?

Reply:
The naked `Place` production corresponds to the `Copy` constructor. I'll add a comment to highlight that.